install/overlays/config.txt

```diff
@@ -0,0 +1,92 @@
+[pi4]
```

Comment:
Maybe the wiki is a better place for it ?

Reply:
done